app.js

```diff
@@ -16,20 +16,7 @@ app.use(function(req, res, next){
 // Logging
 var config = require("./config.js");
 
-// enable detailed API logging without logging JWT
-if (config.logLevel === "debug") {
-    expressWinston.requestWhitelist.push("body");
-    expressWinston.responseWhitelist.push("body");
-} else {
-    expressWinston.requestWhitelist = ["url", "method", "httpVersion", "originalUrl", "query"];
-    expressWinston.responseWhitelist = ["statusCode", "responseTime"];
-}
-app.use(expressWinston.logger({
-    winstonInstance,
-    meta: true, // optional: log meta data about request (defaults to true)
-    msg: "HTTP {{req.method}} {{req.url}} {{res.statusCode}} {{res.responseTime}}ms",
-    colorStatus: true // Color the status code (default green, 3XX cyan, 4XX yellow, 5XX red).
-}));
+
```

Comment:
Why move this?

Reply:
Causes crashes for error responses. It's been fixed for most of the other services but not this one